### PR TITLE
Added missing character

### PR DESCRIPTION
**Pull request by rtisma**

### CODE REVIEW

**Comment by rtisma** [comment]
@caravinci @joneubank

**Review by joneubank — APPROVED** [review]

**Comment by joneubank** [comment]
@rtisma Thanks!

**Comment by andricDu** [comment]
@joneubank `mvn versions:set -DnewVersion=<NEW_VERSION>` in the future to avoid these typos.

File: song-server/pom.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
         <dependency>
             <groupId>bio.overture</groupId>
-            <artifactId>song-core</artifactId
+            <artifactId>song-core</artifactId>
```
Comment (caravinci) [inline song-server/pom.xml:40]: Nice catch!?

**Pull request merged**